tests/ignite/contrib/metrics/regression/test__base.py

```diff
@@ -37,3 +39,27 @@ def compute(self):
     with pytest.raises(TypeError, match=r"Input y dtype should be float"):
         y = torch.tensor([1, 1])
         m.update((y.float(), y))
+
+
+def test_torch_median():
+    n_iters = 80
+    size = 105
+    y_true = torch.rand(size=(n_iters * size,))
+    y_pred = torch.rand(size=(n_iters * size,))
+    e = torch.abs(y_true.view_as(y_pred) - y_pred) / torch.abs(y_true.view_as(y_pred))
+    assert _torch_median(e) == np.median(e)
```

Comment:
```suggestion
@pytest.mark.parametrize("size", [100, 101, (30, 3), (31, 3)])
def test_torch_median(size):
    data = torch.rand(size)
    assert _torch_median(data) == np.median(data.cpu().numpy())
```

Reply:
And rewrite it such that it could be tested on cpu, cuda and xla like here: 
https://github.com/pytorch/ignite/blob/2b7eaf2ab54a0a9a2a6ba65d7531eaff9232a1ab/tests/ignite/engine/test_create_supervised.py#L528-L593